README.md

```diff
@@ -80,9 +80,9 @@ Change directories to the newly cloned repo:
 
      cd cordapp-tutorial
      
-Check out the latest milestone release (currently `release-M8.2`):
+Check out the latest milestone release (currently `release-M9.2`):
```

Comment:
Keep the release number. Developers copy and paste and for developer experience purposes _we_ should maintain this tiny bit of documentation to make it easier for people to get started.